draft-irtf-cfrg-frost.md

```diff
@@ -752,7 +752,8 @@ and publishes the resulting signature.
     return (group_commitment, z)
 ~~~
 
-The output signature (R, z) from the aggregation step MUST be encoded as follows:
+The output signature (R, z) from the aggregation step MUST be encoded as follows
+(using notation from {{Section 3 of TLS}}):
```

Comment:
Should this like to  {{?TLS=RFC8446}} instead?

Reply:
Not necessary since we already did that once before.

Reply:
That is done lower in the document, does the link matching work with one definition anywhere?

Reply:
Yeah, otherwise the document wouldn't build correctly.